### DISCUSSION TITLE
[Question]Newly purchased license can not use pyarmor7 problem

### DISCUSSION THREAD

**Discussion by 130347665** [dashingsoft/pyarmor#1019] [Q&A]
Can I use pyarmor7 with my newly purchased license?
Because my project cannot be upgraded to x64 at the moment
But when I use it now
console will prompt me with
```
INFO PyArmor version 7.7.4
INFO Python 3.8.13
INFO Target platform: local
INFO Get remote file: https://pyarmor.dashingsoft.com/files/r52.6/index.json
ERROR HTTP error 401: not authorized
```

**Comment by jondy** [comment]
The new license could use Pyarmor 7 only in 5 platforms: windows.x86_64, windows.x86, linux.x86_64, linux.x86, Darwin.x86_64. (linux platform must be glibc). 

It's impossible in other platforms to use Pyarmor 7 with new license. 

Pyarmor 8.0+ will support x86 about in 2 months.

**Reply by jondy** [reply]
Pyarmor 8.0 may support x86 in 2 week.

**Comment by 130347665** [comment]
Yes, that's what I'm asking. I'm using x86 python on windows 11 x64, 
so I'm asking why I can't use pyarmor7?
What steps should I add to get my newly purchased license to use pyarmor7?

Otherwise, I use it now and he prompts me.
ERROR HTTP error 401: not authorized
Thank you for your patience in answering

**Reply by jondy** [reply]
Think Pyarmor 8 as another product "pyarmor.cli", it uses all new commands and totally different from pyarmor 7.

Just for old users, Pyarmor 8 still could use old commands of pyarmor 7.

**Comment by joonis** [comment]
Is it possible to register both versions (7+8) side by side?

Version 8 must be installed, but version 7 can run from a folder.

The only issue is to register it at the same time...

**Reply by jondy** [reply]
Use option `--home` to register pyarmor in different path, version 7 and 8 could work in one machine.
